src/accelerate/tracking.py

```diff
@@ -681,12 +686,171 @@ def finish(self):
+                clearml_logger.report_single_value(name=k, value=v, **kwargs)
+                continue
+            title, series = ClearMLTracker._get_title_series(k)
+            clearml_logger.report_scalar(title=title, series=series, value=v, iteration=step, **kwargs)
+
+    @on_main_process
+    def log_images(self, values: dict, step: Optional[int] = None, **kwargs):
+        """
+        Logs `images` to the current run.
+
+        Args:
+            values (`Dict[str, List[Union[np.ndarray, PIL.Image]]`):
+                Values to be logged as key-value pairs. The values need to have type `List` of `np.ndarray` or
+            step (`int`, *optional*):
+                The run step. If included, the log will be affiliated with this step.
+            kwargs:
+                Additional key word arguments passed along to the `clearml.Logger.report_image` method.
+        """
+        clearml_logger = self.task.get_logger()
+        for k, v in values.items():
+            title, series = ClearMLTracker._get_title_series(k)
+            clearml_logger.report_image(title=title, series=series, iteration=step, image=v, **kwargs)
+
+    @on_main_process
+    def log_table(
+        self,
+        table_name: str,
+        columns: List[str] = None,
+        data: List[List[Any]] = None,
+        dataframe: Any = None,
+        step: Optional[int] = None,
+        **kwargs,
+    ):
+        """
+        Log a Table to the task. Can be defined eitherwith `columns` and `data` or with `dataframe`.
+
+        Args:
+            table_name (`str`):
+                The name of the table
+            columns (list of `str`, *optional*):
+                The name of the columns on the table
+            data (List of List of Any data type, *optional*):
+                The data to be logged in the table
+            dataframe (Any data type, *optional*):
+                The data to be logged in the table
+            step (`int`, *optional*):
+                The run step. If included, the log will be affiliated with this step.
+            kwargs:
+                Additional key word arguments passed along to the `clearml.Logger.report_table` method.
+        """
+        if dataframe is None:
+            if not is_pandas_available():
+                raise ImportError(
+                    "`ClearMLTracker.log_table` requires `pandas` be installed. Please fix this by running `pip install pandas`"
+                )
+            import pandas as pd
+
+            if columns is None or data is None:
+                raise ValueError(
+                    "`ClearMLTracker.log_table` requires that `columns` and `data` to be supplied if `dataframe` is `None`"
+                )
+            dataframe = pd.DataFrame(data, columns=columns)
+        title, series = ClearMLTracker._get_title_series(table_name)
+        self.task.get_logger().report_table(title=title, series=series, table_plot=dataframe, iteration=step, **kwargs)
```

Comment:
ClearML [docs](https://clear.ml/docs/latest/docs/references/sdk/logger#report_table) say that:

> Pandas DataFrame or Table as list of rows (list)

are supported. Not sure what they mean by "Table", but if it's just a list of rows, can we get by without pandas here?

Reply:
You are correct :+1: . I made some changes to remove pandas as a req.